snuba/manual_jobs/redis.py

```diff
@@ -72,6 +72,8 @@ def _get_job_types_multi(job_ids_keys: Sequence[str]) -> List[str]:
 
 
 def _get_job_status_multi(job_ids_keys: Sequence[str]) -> List[JobStatus]:
+    if len(job_ids_keys) == 0:
+        return []
```

Comment:
nit: add a newline under this `return []` or make it `return [] if len(job_ids_keys) == 0` (with a newline before the next return)